### PR TITLE
As 1631 interactive cli

### PR DESCRIPTION
**Pull request by mlikasam-askui**

### CODE REVIEW

File: packages/askui-nodejs/package.json [hunk 1]
```diff
@@ -66,6 +69,8 @@
     },
     "devDependencies": {
         "@types/fs-extra": "9.0.13",
+        "@types/listr": "0.14.4",
+        "@types/inquirer": "^9.0.3",
```
Comment (mlikasam-askui) [inline packages/askui-nodejs/package.json:73]:
```suggestion
        "@types/inquirer": "9.0.3",
```

File: packages/askui-nodejs/package.json [hunk 2]
```diff
@@ -85,4 +90,4 @@
     "lint-staged": {
         "./**/*.{js,ts}": "eslint --cache --fix --max-warnings 0"
     }
-}
+}
```
Comment (mlikasam-askui) [inline packages/askui-nodejs/package.json:93]: nl

File: packages/askui-nodejs/bin/askui [hunk 3]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../dist/cjs/lib/copy-example-project').init();
+require('../dist/cjs/lib').init();
```
Comment (mlikasam-askui) [inline packages/askui-nodejs/bin/askui:3]: nl

**Comment by JohannesDienst-askui** [comment]
Heyho @mlikasam-askui could you do a review of the code?

**Pull request closed without merging**